### ISSUE TITLE
ODT->PDF Conversion: Page-Spanning table rows are moved to the next page rather than staying spanned.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#193]
```
>>> What steps will reproduce the problem?
1. Convert the attached "spanning_table_rows.odt" file to PDF using the 
PdfConverter XDocReport class.

>>> What is the expected output? What do you see instead?
The expected output should be what the ODT file looks like: the large, spanned, 
table column with text should span the pages. The PDF drops the table row to 
the next page.

>>> What version of the product are you using? On what operating system?
1.0.0-SNAPSHOT, latest code from the git repo (as of this morning.)
Windows 7 64-bit

```

Original issue reported on code.google.com by `PhilipDN...@gmail.com` on 9 Nov 2012 at 7:27

Attachments:
- [spanning_table_rows.odt](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-193/comment-0/spanning_table_rows.odt)
- [spanning_table_rows.pdf](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-193/comment-0/spanning_table_rows.pdf)

**Comment by GoogleCodeExporter** [comment]
```
@Leszek : I have add a JUnit 
http://code.google.com/p/xdocreport/source/browse/thirdparties-extension/org.odf
toolkit.odfdom.converter.pdf/src/test/resources/org/odftoolkit/odfdom/converter/
core/Issue193.odt with the odt attached of this issue.
```

Original comment by `angelo.z...@gmail.com` on 10 Nov 2012 at 2:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 1.0.0

There is a problem here.
In open office "split row" is "per row" property. In iText it is "per table" 
property, so it can't be exactly translated to iText equivalent.
I introduced a rule -> if any row cannot be splitted than the whole table 
cannot be splitted. This should work properly most times.


```

Original comment by `abe...@gmail.com` on 13 Dec 2012 at 8:19
- Changed state: **Fixed**
